implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -74,6 +70,20 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
         provider.mpJwtPublicKey = Optional.of(publicKey);
         provider.mpJwtLocation = Optional.of(publicKeyLocation);
         provider.mpJwtIssuer = issuer;
+
```

Comment:
Provide same optional defaults given by CDI/MP Config to avoid use of null checks elsewhere in code

Reply:
This will likely break Thorntail